app.js

```diff
@@ -34,3 +34,52 @@ var stuff = [
 //   // console.log(stuff[i].name);
 //   // console.log(stuff[i].src);
 // }
+
+function findRandom(){
+  var tempArray = [];
+  while(tempArray.length < 3){
+    var rand = stuff[Math.floor(Math.random() * stuff.length)].src;
+    if(tempArray.indexOf(rand) === -1){
+      tempArray.push(rand);
+    }
+    // console.log(tempArray);
+  }
+  randomNumArray = tempArray;
+}
+findRandom();
+console.log(randomNumArray);
+
+var pictureBlockEl = document.getElementById('productPictures');
+function postPictures(){
+  var pictureEl = document.createElement('h1');
+  pictureBlockEl.appendChild(pictureEl);
+
+  for(var i = 0; i < randomNumArray.length; i++){
+    var addPic = document.createElement('img');
+    addPic.setAttribute('src', randomNumArray[i]);
+    pictureEl.appendChild(addPic);
+
+    console.log(typeof(randomNumArray[i]));
+  }
+}
+
+function removePictures(){
+  var pictureEl = document.getElementById('prodcutPictures');
+  var removePicEl = document.getElementByTag('h1');
+  pictureEl.removeChild(removePicEl);
+}
+postPictures();
+
+console.log('-------------------Event Listener click---------------');
+
+var onImageClickEl = document.getElementById('productPictures');
+
+onImageClickEl.addEventListener('click', handleClick);
+
+//event listener not working corectly yet. Needs to delete current picure and add new
+function handleClick(event){
```

Comment:
This is a great start! Commenting it like that, and breaking down the problem into more manageable little parts, is really smart!